dist/s_fixme

```diff
@@ -0,0 +1,25 @@
+#! /bin/bash
+
+# Find FIXME comments related to the ticket potentially indicated by the branch name.
+# The script only fails if the branch name follows a specific syntax and FIXME comments are found,
+# otherwise, the script exits without failing.
+
+# Retrieve the current branch name.
+branch_name=$(git rev-parse --abbrev-ref HEAD)
```

Comment:
Random suggestion: $(git rev-parse --abbrev-ref HEAD 2>&1 || echo FAILURE) so as to not print git messages to stderr if run outside a repository, and also to avoid matching if some other unexpected failure occurs.

Reply:
Good idea, thanks! Done in 4c65895a8232a6df3147f1bce46a59c46ab3e5d7.